css/styles.css

```diff
@@ -1,6 +1,29 @@
 /*<--------------- IDs ---------------->*/
 #container {
-  border: 1px solid black; }
+  border: 1px solid black; 
+  padding: 160px 30px 30px 30px;
```

Comment:
This css works, but it's not in the scss file, so once someone runs grunt, it overrides the css. try moving all this to the scss file as well?